### ISSUE TITLE
Broken types in v3

### ISSUE THREAD

**Issue by chybisov** [gorhom/react-native-bottom-sheet#280]
# Bug

Types are broken after upgrading to _v3.3.1_. I assume they should be optional.

![image](https://user-images.githubusercontent.com/18644653/107747281-da5eed00-6d1f-11eb-8fba-ce215c14fc1e.png)

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | 3.3.1   |
| react-native                    | 0.63.4   |
| react-native-reanimated         | 2.0.0-rc0   |
| react-native-gesture-handler    | 1.10.0   |
| typescript    | 4.1.5   |

## Steps To Reproduce

1. Add BottomSheet to project with typescript.

Describe what you expected to happen:

1.  Above props should be optional

**Comment by SleeplessByte** [comment]
Here is the patch-package diff to fix this for 3.4.1. Feel free to use, feel free to PR (and even take credit).

```diff
diff --git a/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts b/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
index 2d2439c..0d96791 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
+++ b/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
@@ -8,7 +8,7 @@ import type { BottomSheetBackgroundProps } from '../bottomSheetBackground';
 
 export interface BottomSheetProps
   extends BottomSheetAnimationConfigs,
-    Pick<
+    Partial<Pick<
       PanGestureHandlerProps,
       | 'activeOffsetY'
       | 'activeOffsetX'
@@ -16,7 +16,7 @@ export interface BottomSheetProps
       | 'failOffsetX'
       | 'waitFor'
       | 'simultaneousHandlers'
-    > {
+    >> {
   // configuration
   /**
    * Initial snap index, you also could provide `-1` to initiate bottom sheet in closed state.
```

**Comment by gorhom** [comment]
thanks @SleeplessByte , @chybisov, this should be resolved with `v3.6.2`
